plugins/tweak/tweak.cpp

```diff
@@ -329,9 +336,20 @@ DFhackCExport command_result plugin_init (color_ostream &out, std::vector <Plugi
 
     TWEAK_HOOK("nestbox-color", nestbox_color_hook, drawBuilding);
 
+    TWEAK_HOOK("partial-items", partial_items_hook_1, getItemDescription);
```

Comment:
I would prefer `partial_items_hook_bar`, `_drink`, `_glob`, etc., instead of `_1`, `_2`, `_3`.

Reply:
+1 on this. I was thinking the same thing as I read the code.